dbms/src/Flash/tests/gtest_topn_executor.cpp

```diff
@@ -214,5 +245,25 @@ try
 }
 CATCH
 
+TEST_F(ExecutorTopNTestRunner, BigTable)
+try
+{
+    std::vector<String> tables{"big_table_1", "big_table_2", "big_table_3"};
+    for (const auto & table : tables)
+    {
+        std::vector<size_t> limits{0, 1, 10, 20, 199, 200, 300};
+        for (auto limit_num : limits)
+        {
+            auto request = context
+                               .scan("test_db", table)
+                               .topN("key", false, limit_num)
+                               .build(context);
+            auto expect = executeStreams(request, 1);
```

Comment:
more concurrency?

Reply:
Various concurrency and block sizes have already been executed in https://github.com/pingcap/tiflash/blob/957d2d41c07d538edab9e47bdbb7eaede753c60b/dbms/src/TestUtils/ExecutorTestUtils.cpp#L105-L146.
Here just to get a baseline of the results.

Reply:
> 

okk